runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContentsClientBridge.java

```diff
@@ -247,6 +247,11 @@ public void onProgressChanged(int progress) {
     }
 
     @Override
+    public void onOverScrolled(int scrollX, int scrollY, boolean clampedX, boolean clampedY) {
+        mXWalkView.onOverScrolledDelegate(scrollX, scrollY, clampedX, clampedY);
+    }
+
+    @Override
```

Comment:
For the event is bottom up. I nned to call onOverScrolledDelegate.
And for the WebView onOverScrolled API, it will be called whenever a scroll event happens. so it is not only binding for overscroll event. To keep the API behavior the same as WebView. I add onOverScrolledDelegate call in onScrollChanged.